demo/diagram-page.css

```
@@ -0,0 +1,317 @@
+}
+
+body {
+  --t-bg: #ffffff;
+  --t-fg: #27272a;
+  --t-accent: #3b82f6;
+  --t-brand: #9570be;
+  --t-muted: color-mix(in srgb, var(--t-fg) 55%, var(--t-bg));
+  --t-border: color-mix(in srgb, var(--t-fg) 12%, var(--t-bg));
+
+  font-family:
+    'Geist',
```

Stylelint (17.14.0)

[error] Expected no quotes around "Geist" (font-family-name-quotes)

(font-family-name-quotes)